backend/src/test/kotlin/ch/learntrack/backend/backoffice/user/UserIntegrationTest.kt

```diff
@@ -165,4 +165,21 @@ class UserIntegrationTest: IntegrationTest() {
                 .isEqualTo(HttpStatus.CONFLICT)
     }
 
+    @Test
+    fun `should be able to create a user even if email is in upper case`() {
```

Comment:
```suggestion
    fun `should save uppercase email as lowercase on user creation`() {
```
or something like that as it describes more what you are testing here.